homeassistant/components/reolink/binary_sensor.py

```diff
@@ -121,6 +137,140 @@ class ReolinkBinarySensorEntityDescription(
     ),
 )
 
+BINARY_SMART_AI_SENSORS = (
+    ReolinkSmartAIBinarySensorEntityDescription(
+        key="crossline",
+        ai_type="people",
+        cmd_id=33,
+        translation_key="crossline_person",
+        value=lambda api, ch, loc: (
+            api.baichuan.smart_ai_state(ch, "crossline", loc, "people")
+        ),
+        supported=lambda api, ch, loc: (
+            api.supported(ch, "ai_crossline")
+            and "people" in api.baichuan.smart_ai_type_list(ch, "crossline", loc)
+        ),
+    ),
+    ReolinkSmartAIBinarySensorEntityDescription(
+        key="crossline",
```

Comment:
should we rename `key` to something else, since we have multiple Descriptions with the same `key`? I think `key` is usually unique.